modules/Router.js

```diff
@@ -82,7 +84,10 @@ const Router = React.createClass({
       history,
       createRoutes(routes || children)
     )
+
     const router = createRouterObject(history, transitionManager)
+    router.location = null
+    router.params = null
```

Comment:
Not a big fan of duplication but it doesn’t seem harmful yet. Can extract to a method later if necessary.